apps/code-examples/src/app/code-examples/ag-grid/data-grid/data-manager-multiselect/data-manager-multiselect-data-grid-docs-demo-view-grid.component.ts

```diff
@@ -0,0 +1,302 @@
+      sort: 'asc',
+    },
+    {
+      field: 'endDate',
+      headerName: 'End date',
+      type: SkyCellType.Date,
+      valueFormatter: this.endDateFormatter,
+    },
+    {
+      field: 'department',
+      headerName: 'Department',
+      type: SkyCellType.Autocomplete,
+    },
+    {
+      field: 'jobTitle',
+      headerName: 'Title',
+      type: SkyCellType.Autocomplete,
+    },
+  ];
+
+  private columnPickerOptions: SkyDataManagerColumnPickerOption[] = [
+    {
+      id: 'selected',
+      label: '',
+      alwaysDisplayed: true,
+    },
+    {
+      id: 'name',
+      label: 'Name',
+      description: 'The name of the employee.',
+    },
+    {
+      id: 'age',
+      label: 'Age',
+      description: 'The age of the employee.',
+    },
+    {
+      id: 'startDate',
+      label: 'Start date',
+      description: 'The start date of the employee.',
+    },
+    {
+      id: 'endDate',
+      label: 'End date',
+      description: 'The end date of the employee.',
+    },
+    {
+      id: 'department',
+      label: 'Department',
+      description: 'The department of the employee',
+    },
+    {
+      id: 'jobTitle',
+      label: 'Title',
+      description: 'The job title of the employee.',
+    },
+  ];
+
+  public dataState = new SkyDataManagerState({});
+
+  public columnApi?: ColumnApi;
+  public displayedItems: any[] = [];
+  public gridApi?: GridApi;
+  public isGridReadyForInitialization = false;
+  public gridOptions!: GridOptions;
+  public noRowsTemplate: string;
+  public isThisViewActive = false;
+
+  public viewConfig: SkyDataViewConfig = {
+    id: this.viewId,
+    name: 'Data Grid View',
+    icon: 'table',
+    searchEnabled: true,
+    sortEnabled: true,
+    multiselectToolbarEnabled: true,
+    columnPickerEnabled: true,
+    filterButtonEnabled: true,
+    columnOptions: this.columnPickerOptions,
+  };
+
+  constructor(
+    private skyAgGridService: SkyAgGridService,
+    private changeDetector: ChangeDetectorRef,
+    private dataManagerService: SkyDataManagerService
+  ) {
+    this.noRowsTemplate = `<div class="sky-deemphasized">No results found.</div>`;
+  }
+
+  public ngOnInit(): void {
+    this.displayedItems = this.items;
+
+    this.dataManagerService.initDataView(this.viewConfig);
+
+    this.gridOptions = this.skyAgGridService.getGridOptions({
+      gridOptions: {
+        columnDefs: this.multiSelectColumnDefinitions,
+        rowSelection: 'multiple',
+        onGridReady: this.onGridReady.bind(this),
+      },
+    });
+
+    this.dataManagerService
+      .getDataStateUpdates(this.viewId)
+      .subscribe((state) => {
+        this.dataState = state;
+        this.updateColumnOrder();
+        this.isGridReadyForInitialization = true;
+        this.updateDisplayedItems();
+        this.changeDetector.detectChanges();
+      });
+
+    this.dataManagerService.getActiveViewIdUpdates().subscribe((id) => {
+      this.isThisViewActive = id === this.viewId;
```

Comment:
lets rename to just `isViewActive`

Reply:
fixed in: [rename properties](https://github.com/blackbaud/skyux/pull/531/commits/807ce4513c477a7543e9ba9cca8700aa21ba2365)